components/event-publisher-proxy/pkg/handler/beb/handler_integration_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build integration
```

Comment:
I discussed this test with @k15r recently. I think this is a component test.
The reason why I think so is that this test basically tests the whole EPP component by:
- starting the EPP with a handler and message sender
- sending message to the EPP
- checking the return code of the EPP (there is even a real nats server behind the scenes).

I think this definition of a component test matches here:
> a component test is a test that takes the whole component, isolates it as much as possible from its dependencies (by replacing them with mocks and stubs if possible) and then tests how the component reacts to different stimuli in the test suite.

Why do you think this is only an integration test ? We could also discuss this with @ameteiko 🤔